src/hashtables.jl

```diff
@@ -26,11 +27,13 @@ end
 function show_params_TaylorN()
     info( """`TaylorN` and `HomogeneousPolynomial` parameters:
     Maximum order       = $(_params_taylorN.maxOrder)
-    Number of variables = $(_params_taylorN.numVars)""" )
+    Number of variables = $(_params_taylorN.numVars)
+    Variable names = $(_params_taylorN.variable_names)
+    """)
     nothing
 end
 
-global const _params_taylorN = ParamsTaylorN(6,2)
+global const _params_taylorN = ParamsTaylorN(6,2, ["x", "y", "z", "t"])
```

Comment:
There is an unexported `set_variables` method to change the names. This all needs a lot more work but I think the idea is good. 

I see what you mean about the number of variables. This was really just trying out some ideas and I shouldn't really have made it a PR yet. But I think I have a good idea about what to do, along the lines of `SymPy`:

x, y = variables("x y")

should fix both the names *and* the number of variables. (I.e. it should automatically change the current number of variables.) Does this sound good to you?